drizzlepac/haputils/diagnostic_json_harvester.py

```diff
@@ -205,11 +208,17 @@ def json_harvester(json_search_path=os.getcwd(), log_level=logutil.logging.INFO,
     log.setLevel(log_level)
 
     # Get sorted list of json files
-    json_dict = get_json_files(search_path=json_search_path, log_level=log_level)
+    json_dict = get_json_files(search_path=json_search_path,
+                               search_patterns=json_patterns,
+                               log_level=log_level)
     master_dataframe = None
     # extract all information from all json files related to a specific Pandas DataFrame index value into a
     # single line in the master dataframe
-    for idx in json_dict.keys():
+    num_json = len(json_dict)
+    for n,idx in enumerate(json_dict.keys()):
+        if ((n/num_json) % 0.1) == 0:
+            log.info("Harvested {}% of the JSON files".format((n/num_json)*100))
+
```

Comment:
This progress information is very nice to have!

Reply:
This minimizes the log messages while still providing progress on the processing.  Any time stamps on the log messages themselves then act as timing for the processing.  It skips some values, since it relies on exact match to 0.1, but it helps.